source/includes/driver-compatibility-caveat.rst

```diff
@@ -0,0 +1,6 @@
+.. note::
+
+   The compatibility matrices show which versions of the driver are
+   recommended for use with a given major release. Older driver versions
+   may be able to connect to a version of MongoDB which is not officially
+   supported, but not all features will be available.
```

Comment:
I single-sourced it because Shane and I discussed the possibility of including the caveat paragraph on each of the individual driver pages and I wanted to leave the possibility open, but if we're only going to use it on the one page (the driver compatibility reference) I can inline it.